autogluon/utils/tabular/contrib/tabular_nn_pytorch/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,4 @@
+def get_default_searchspace(problem_type, num_classes=None):
```

Comment:
Can we have a default search space for HPO?  I think since it's non-default, the users who try this model may likely be interested in hyperparameter-tuning it as well.

Reply:
added